lib/readline.js

```diff
@@ -921,6 +921,8 @@ exports.Interface = Interface;
 const KEYPRESS_DECODER = Symbol('keypress-decoder');
 const ESCAPE_DECODER = Symbol('escape-decoder');
 
+const ESCAPE_CODE_TIMEOUT = 500;
```

Comment:
Seems a bit high? Maybe add a comment as to the rough reasoning behind the value